### PR TITLE
docs: update readme for diataxis structure

### PR DESCRIPTION
**Pull request by lucasmoura**
## Proposed Commit Message
docs: update readme for diataxis structure

Small readme update for diataxis structure

## Desired commit type

 - [ ] Squash and merge: Using the "Proposed Commit Message"
 - [ ] Create a merge commit and use "Proposed Commit Message"
 - [x] Rebase and merge, no merge commit, rely only on existing commit messages

## Checklist

 - [ ] I have updated or added any unit tests accordingly
 - [ ] I have updated or added any integration tests accordingly
 - [x] I have updated or added any documentation accordingly

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,9 +15,10 @@
 ![Released Focal Version](https://img.shields.io/ubuntu/v/ubuntu-advantage-tools/focal?label=Focal&logo=ubuntu&logoColor=white)
 ![Released Jammy Version](https://img.shields.io/ubuntu/v/ubuntu-advantage-tools/jammy?label=Jammy&logo=ubuntu&logoColor=white)
 
-The Ubuntu Advantage (UA) Client provides users with a simple mechanism to
-view, enable, and disable offerings from Canonical on their system. The
-following entitlements are supported:
+The Ubuntu Advantage (UA) Client is the offical plataform to enable Canonical offering on your
```
Comment (orndorffgrant) [inline README.md:18]:
```suggestion
The Ubuntu Advantage (UA) Client is the official platform to enable Canonical offerings on your
```

Comment (orndorffgrant) [reply README.md:18]: Also "platform" kind of implies something bigger/broader than what we offer I think. Maybe replace "platform" with just "tool"

File: README.md [hunk 2]
```diff
@@ -27,12 +28,13 @@ following entitlements are supported:
...
-## Quickstart
 
-`ua` is already installed on every Ubuntu system. Try it out by running `ua help`!
+If you need any of those services in your machine, UA is the right tool for you.
+Furthermore, UA is already installed on every Ubuntu system. Try it out by running `ua help`!
 
-To learn the basic commands and enable professional Ubuntu services in a sandbox environment, follow our [Getting Started Tutorial](./docs/tutorials/basic_ua_commands.md).
+## Quickstart
 
+To learn the basic commands and enable professional Ubuntu services in a sandbox environment, follow our [Getting Started Tutorial](./docs/tutorials/basic_ua_commands.md).
```
Comment (orndorffgrant) [inline README.md:37]: I added this, but now thinking it doesn't add much value over the first entry in the tutorials list. We can just remove it
```suggestion
```

File: README.md [hunk 3]
```diff
@@ -27,12 +28,13 @@ following entitlements are supported:
 - [FIPS 140-2 Certified Modules (and optional non-certified patches)](https://ubuntu.com/security/fips)
 - [Livepatch Service](https://ubuntu.com/security/livepatch)
 
-## Quickstart
 
-`ua` is already installed on every Ubuntu system. Try it out by running `ua help`!
+If you need any of those services in your machine, UA is the right tool for you.
```
Comment (orndorffgrant) [inline README.md:32]:
```suggestion
If you need any of those services for your machine, UA is the right tool for you.
```

**Review by orndorffgrant — COMMENTED** [review]
Nice! Just some nits

**Comment by lucasmoura** [comment]
@orndorffgrant updated

**Review by orndorffgrant — APPROVED** [review]

**Pull request merged**